write customise docs (#35)
did-app/www: docs/customise-authentication-pages/index.md

---
layout: "doc"
title: "Customise authentication pages"

> abstract: "Add custom messaging and color schemes to the DID.app sign in."

Reviewer: ```suggestion
abstract: "Add custom messaging and colour schemes to the DID.app sign in."
```

---

*Upgrade to our startup tier, or higher, to start customising your sign in pages.*

To edit the customisation options for an app, visit the admin page for that app.

> You will see the following options if there are available.

Reviewer: ```suggestion
You will see the following options if they are available:
```

PICTURE of options

> You can set a color to match your brand color scheme, this will be usef on all the buttons and highlights in the sign in widget.

Reviewer: ```suggestion
You can set a colour to match your brand colour scheme, this will be used on all the buttons and highlights in the sign in widget.
```

An optional welcome message can also be added, this can help set a feel for your application.
Keep it light or serious it's up to you.

PICTURE of customised screens 

## Custom UI (Coming Soon)

DID.app's sign in widget is carefully designed to be clear and easy for your users to navigate.

> This focus on a simple clear design gets your users authenticated and using your site quicker.

Reviewer: ```suggestion
This focus on a simple clear design gets your users authenticated and using your site faster.
```

> If you need more control you can direct with our widget API directly, this allows you to capture a users email address any way you want.

Reviewer: ```suggestion
If you need more control you can call our widget API directly.  This allows you to capture a user's email address any way you want and in so doing create a custom UI.
```

> Get in touch with `team@did.app` if you want to find out more.

Reviewer: ```suggestion
This feature is coming soon. Please get in touch with `team@did.app` if you want to find out more.
```
